### PR TITLE
Add view command

### LINKED ISSUE #49: Add view command

### PR DESCRIPTION
**Pull request by mgiang2015**
Allow user to view only the book specified by index. No additional information is shown compared to list command for now, will see how that can be improved
closes #49

### CODE REVIEW

File: src/test/java/seedu/bookmark/logic/commands/ViewCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        Index outOfBoundIndex = Index.fromOneBased(model.getFilteredBookList().size() + 1);
+        ViewCommand viewCommand = new ViewCommand(outOfBoundIndex);
+
+        assertCommandFailure(viewCommand, model, Messages.MESSAGE_INVALID_BOOK_DISPLAYED_INDEX);
+    }
+
+    @Test
+    public void equals() {
+        ViewCommand deleteFirstCommand = new ViewCommand(INDEX_FIRST_BOOK);
+        ViewCommand deleteSecondCommand = new ViewCommand(INDEX_SECOND_BOOK);
```
Comment (TanLeYang) [inline src/test/java/seedu/bookmark/logic/commands/ViewCommandTest.java]: I think there's a typo here, should be `viewFirstCommand` and `viewSecondCommand` right?

File: src/main/java/seedu/bookmark/model/book/IsSpecifiedBookPredicate.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package seedu.bookmark.model.book;
+
+import java.util.function.Predicate;
+
+public class IsSpecifiedBookPredicate implements Predicate<Book> {
```
Comment (TanLeYang) [inline src/main/java/seedu/bookmark/model/book/IsSpecifiedBookPredicate.java]: Perhaps you could include a super simple javadoc  for this class?

**Review by TanLeYang — COMMENTED** [review]
Looks good except for some small stuffs! Fix those and it'll be good to merge 👍

**Review by pennhanlee — COMMENTED** [review]
Besides Leyang's comments, LGTM!

**Review by pangpuncake — APPROVED** [review]
LGTM!

**Pull request merged**